bentoml/_internal/frameworks/common/pytorch.py

```diff
@@ -54,8 +54,8 @@ def __init__(self, *inner_args: t.Any, **inner_kwargs: t.Any) -> None:
 
 
 class PytorchModelRunnable(bentoml.Runnable):
-    SUPPORT_NVIDIA_GPU = True
-    SUPPORT_CPU_MULTI_THREADING = True
+    SUPPORTED_RESOURCES = ("nvidia.com/gpu",)
```

Comment:
shall we have "cpu" in it by default?

Reply:
Hm, that's a good point. We're kind of special casing the strategies around CPU right now but it definitely wouldn't hurt to include `"cpu"` everywhere.

Reply:
Yeah it make users even easier to understand